### PR TITLE
fix: next.json

### PR DESCRIPTION
**Pull request by akira-toriyama**
Corrected to `json` generated using `create next-app --typescript`.

Sorry for the many Pull Request. 🫥

### CODE REVIEW

File: bases/next.json [hunk 1]
```diff
@@ -6,15 +6,17 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": false,
```
Comment (akira-toriyama) [inline bases/next.json:9]: https://nextjs.org/docs/api-reference/create-next-app

`create next-app --typescript ` was used to generate the data.

File: bases/next-strictest.json [hunk 2]
```diff
@@ -20,12 +20,14 @@
     "importsNotUsedAsValues": "error",
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
-    "incremental": true,
```
Comment (akira-toriyama) [inline bases/next-strictest.json:23]: Moved to the same location as `create next-app --typescript`.

File: bases/next.json (hunk 1 above)
Comment (akira-toriyama) [reply bases/next.json:9]: In the latest `tsconfig`, `"strict": true` 😰

File: bases/next.json [hunk 3]
```diff
@@ -6,15 +6,17 @@
...
-    "incremental": true,
     "esModuleInterop": true,
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
-  }
+    "jsx": "preserve",
+    "incremental": true
```
Comment (akira-toriyama) [inline bases/next.json:18]: Moved to the same location as `create next-app --typescript`.

File: bases/next-strictest.json [hunk 4]
```diff
@@ -20,12 +20,14 @@
...
     "esModuleInterop": true,
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
-  }
+    "jsx": "preserve",
+    "incremental": true
```
Comment (akira-toriyama) [inline bases/next-strictest.json:29]: Moved to the same location as `create next-app --typescript`.

**Comment by orta** [comment]
Looks like it is moving to strict true: https://github.com/vercel/next.js/blob/v13.0.0/packages/create-next-app/tsconfig.json

**Pull request closed without merging**